.environment/soap_service/soap/response/v2/sjIvfk.res

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<soapMockResponse>
+    <body>&lt;soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:web="http://nedss.state.pa.us/2012/B01/elrwcf"&gt;
+   &lt;soapenv:Header/&gt;
+   &lt;soapenv:Body&gt;
+      &lt;web:UploadFilesResponse&gt;?&lt;/web:UploadFilesResponse&gt;
+   &lt;/soapenv:Body&gt;
+&lt;/soapenv:Envelope&gt;</body>
+    <contentEncodings/>
+    <httpHeaders/>
+    <httpStatusCode>200</httpStatusCode>
+    <id>sjIvfk</id>
+    <name>Auto-generated mocked response</name>
+    <operationId>6j63DO</operationId>
+    <status>ENABLED</status>
+    <usingExpressions>false</usingExpressions>
+    <xpathExpressions/>
+</soapMockResponse>
```

Comment:
This sets up CastleMock in our dev environment to have a response that mocks PA's SOAP service.